api/schemas/AggregateRegionSummary.json

```diff
@@ -174,6 +174,27 @@
         "ICUBeds"
       ]
     },
+    "Metrics": {
```

Comment:
How do the API schemas get used?

Reply:
They're now used a couple ways.

1. The frontend code actually consumes them to generate typed classes.
2. we use it now to autogenerate the readme documentation